src/stores/SpaceStore.tsx

```diff
@@ -111,6 +114,26 @@ export class SpaceStoreClass extends AsyncStoreWithClient<IState> {
         this.emit(UPDATE_SELECTED_SPACE, this.activeSpace);
         this.emit(SUGGESTED_ROOMS, this._suggestedRooms = []);
 
+        // view last selected room from space
+        const spaceId = space?.roomId || LAST_VIEWED_ROOMS_HOME;
+        const roomId = window.localStorage.getItem(`${LAST_VIEWED_ROOMS}_${spaceId}`);
+
+        if (roomId) {
+            defaultDispatcher.dispatch({
```

Comment:
we should probably check that we are still joined to the room, if we are not then maybe go to space home